json/src/lib.rs

```diff
@@ -1120,6 +1120,196 @@ pub struct GetDescriptorInfoResult {
+    pub rules: Vec<GetBlockTemplateResultRules>,
+    /// List of features the Bitcoin Core getblocktemplate implementation supports
+    pub capabilities: Vec<GetBlockTemplateResultCapabilities>,
+    /// Set of pending, supported versionbit (BIP 9) softfork deployments
+    #[serde(rename = "vbavailable")]
+    pub version_bits_available: HashMap<u32, String>,
+    /// Bit mask of versionbits the server requires set in submissions
+    #[serde(rename = "vbrequired")]
+    pub version_bits_required: u32,
+    /// Id used in longpoll requests for this template.
+    pub longpollid: String,
+    /// List of transactions included in the template block
+    pub transactions: Vec<GetBlockTemplateResultTransaction>,
+    /// The signet challenge. Only set if mining on a signet, otherwise empty
+    #[serde(default, with = "bitcoin::blockdata::script::Script")]
+    pub signet_challenge: bitcoin::blockdata::script::Script,
+    /// The default witness commitment included in an OP_RETURN output of the
+    /// coinbase transactions. Only set when mining on a network where SegWit
+    /// is activated.
+    #[serde(with = "bitcoin::blockdata::script::Script", default)]
+    pub default_witness_commitment: bitcoin::blockdata::script::Script,
+    /// Data that should be included in the coinbase's scriptSig content. Only
+    /// the values (hexadecimal byte-for-byte) in this map should be included,
+    /// not the keys. This does not include the block height, which is required
+    /// to be included in the scriptSig by BIP 0034. It is advisable to encode
+    /// values inside "PUSH" opcodes, so as to not inadvertently expend SIGOPs
+    /// (which are counted toward limits, despite not being executed).
+    pub coinbaseaux: HashMap<String, String>,
+    /// Total funds available for the coinbase
+    #[serde(rename = "coinbasevalue", with = "bitcoin::util::amount::serde::as_sat", default)]
+    pub coinbase_value: Amount,
+    /// The number which valid hashes must be less than, in big-endian
+    #[serde(with = "::serde_hex")]
+    pub target: Vec<u8>,
+    /// The minimum timestamp appropriate for the next block time. Expressed as
+    /// UNIX timestamp.
+    #[serde(rename = "mintime")]
+    pub min_time: u64,
+    /// List of things that may be changed by the client before submitting a
+    /// block
+    pub mutable: Vec<GetBlockTemplateResulMutations>,
+    /// A range of valid nonces
+    #[serde(with = "::serde_hex", rename = "noncerange")]
+    pub nonce_range: Vec<u8>,
+}
+
+/// Models a single transaction entry in the result of "getblocktemplate"
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
+pub struct GetBlockTemplateResultTransaction {
+    /// The transaction id
+    pub txid: bitcoin::Txid,
+    /// The wtxid of the transaction
+    #[serde(rename = "hash")]
+    pub wtxid: bitcoin::Wtxid,
+    /// The serilaized transaction bytes
+    #[serde(with = "::serde_hex", rename = "data")]
+    pub rawtx: Vec<u8>,
```

Comment:
`raw_tx`*

Also a getter `pub transaction(&self) -> Result<bitcoin::Transaction, encode::Error>` would be nice! (Some other structs here also have that.)